package.json

```diff
@@ -23,7 +23,9 @@
 		"test:sast:sandworm": "sandworm-audit --skip-all --show-tips false",
 		"test:sast:semgrep": "semgrep scan --config auto",
 		"test:sast:trufflehog": "trufflehog filesystem --only-verified --log-level=-1 ./",
-		"test:types": "ls packages | xargs -I {} tsd packages/{}",
+		"test:types:tsd": "find packages -name \"*.test-d.ts\" -exec dirname {} \\; | sed 's|packages/||' | sort -u | xargs -I {} tsd packages/{}",
+		"test:types:tstyche": "find packages -name \"*.tst.ts\" -exec dirname {} \\; | sed 's|packages/||' | sort -u | xargs -I {} tstyche packages/{}",
```

Comment:
This one allows us to keep both systems during the migration, and run the tests only where they are available for that framework.